src/main/java/Car.java

```diff
@@ -0,0 +1,25 @@
+public class Car {
+
+    private final String name;
+    private int position;
+    private static final int MOVE_TRIGGER = 4;
+
+    public Car(String name){
+        this.name = name;
+        this.position = 0;
+    }
```

Comment:
```suggestion
    public Car(String name) {
        this.name = name;
        this.position = 0;
    }
```

공백도 컨벤션입니다! IDE의 코드 정렬 단축키를 활용하면 좋을 것 같아요.